src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -454,6 +473,8 @@ public String getPatronGroupIdAtCheckout() {
   }
 
   public Loan renew(ZonedDateTime dueDate, String basedUponLoanPolicyId) {
+    log.debug("renew:: parameters dueDate: {}, basedUponLoanPolicyId: {}",
```

Comment:
`ZonedDateTime#toString` is not that trivial. I would log it via Supplier.

Reply:
done